src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -14,19 +14,72 @@
 
 package com.google.sps.classes;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 // TODO(aabundis): Add JUnit tests for utility functions.
 
 /**
  * Utility methods used across classes. Just import class to access all methods.
  */
 public final class Utility {
+  // Variables needed to connect to MySQL database.
+  static final String URL = "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=PST8PDT";
+  static final String USER = "root";
+  static final String PASSWORD = "";
```

Comment:
nit: Can these be named something like SQL_URL, SQL_USER, SQL_PASSWORD? Just so it's clearer what they're referring to, and so no one mixes them up with other urls/users/passwords.

Reply:
Done!